### PR TITLE
Update README.md with Issue #60

### PR DESCRIPTION
**Pull request by angushtlam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+I could love you just like that
+And I could leave you just this fast
+
+But you don't judge me
+'Cause if you did, baby, I would judge you too
+No, you don't judge me
+'Cause if you did, baby, I would judge you too
+
+'Cause I got issues
+But you got 'em too
```
Comment (davidlohle) [inline README.md:16]: i like this

**Review by davidlohle — APPROVED** [review]
nice nice i like it nice

File: README.md (hunk 1 above)
Comment (angushtlam) [reply README.md:16]: thanks m8 🦉

Comment (angushtlam) [reply README.md:16]: @Proplex omg you wrote minecraft plugins too 💯

Comment (doriskwan) [reply README.md:16]: Perfect

**Pull request merged**